docker/helm/templates/master-statefulset.yaml

```diff
@@ -58,6 +58,8 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      securityContext:
+        fsGroup: {{ .Values.gid }}
```

Comment:
> `fsGroupPolicystring`
> Defines if the underlying volume supports changing ownership and permission of the volume before being mounted. Refer to the specific FSGroupPolicy values for additional details. This field is alpha-level, and is only honored by servers that enable the CSIVolumeFSGroupPolicy feature gate.

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/

Reply:
I think that's ok.